web/documentserver-example/ruby/app/controllers/home_controller.rb

```diff
@@ -278,6 +278,19 @@ def csv
     send_file csvPath, :x_sendfile => true
   end
 
+  # downloading an assets file
+  def asset
```

Comment:
> _s_ is missed, `asset -> assets`.

I think if the route and function have the same name it will be better.

Reply:
Got it. However, we've no backward compatibility requirements, so you can easily rename the method and its endpoint. It's better to do this because it's a grammatical error.

Reply:
@sshakndr, you mark it as resolved, but it isn't.